### PR TITLE
qa/cephfs: add back POOL_APP_NOT_ENABLED to ignorelist (partial revert)

### PR DESCRIPTION
**Pull request by vshankar**
Partially reverts ceph/ceph#53037

### CODE REVIEW

File: qa/cephfs/begin/1-ceph.yaml [hunk 1]
```diff
@@ -3,4 +3,3 @@ log-rotate:
   ceph-osd: 10G
 tasks:
   - ceph:
-      create_rbd_pool: false
```
Comment (idryomov) [inline qa/cephfs/begin/1-ceph.yaml:6]: This should be kept, there is no reason to create an RBD pool for CephFS tests.

Comment (vshankar) [reply qa/cephfs/begin/1-ceph.yaml:6]: oh yes, I didn't add the source while pushing the signed-off commit.

Comment (vshankar) [reply qa/cephfs/begin/1-ceph.yaml]: I resorted to reverting just the one commit from PR #53037.

**Comment by vshankar** [comment]
> Partially reverts #53037

**Review by idryomov — APPROVED** [review]
I know @batrick would rather see a revert of https://github.com/ceph/ceph/pull/47560 and I brought up the same on the dev list, although for different reasons.  @batrick appears to want to this health alert to be disabled in testing universally rather than in each suite individually, whereas I maintain that the health alert itself should be fixed so that it "behaves" for regular users/operators too:
> It shouldn't be just developers running their tests with a special knob set who get the sane behavior.

Alas, neither of us has gotten much traction...

**Comment by vshankar** [comment]
> I know @batrick would rather see a revert of #47560 and I brought up the same on the dev list, although for different reasons. @batrick appears to want to this health alert to be disabled in testing universally rather than in each suite individually,

I should have mentioned here that in stand-up, a couple of days back, @batrick agreed to move forward with the revert of the revert (this change) till discussions reach a conclusion on #47560.

**Comment by batrick** [comment]
https://github.com/ceph/ceph/pull/53336

**Review by batrick — APPROVED** [review]

**Pull request merged**